### DISCUSSION TITLE
Embedd hub room without cookies

### DISCUSSION THREAD

**Discussion by ngmiduc** [mozilla/hubs#5281] [General Questions & Discussion]
Is there a way to embedd a Room on a website without Mozilla setting a cookie ?
When I embedd a room, it puts a GA cookie from hub.mozilla.com on my webpage. 

The problem is that it might be problematic for the European Data Protection Law.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/HUBS-1133)

**Comment by normanwankowski** [comment]
I have the same problem. But actually there must be a way to add a cookie notice script in my case.

**Comment by ngmiduc** [comment]
Yes adding a cockie notice would be good or having a non-cookie embedd.

**Comment by normanwankowski** [comment]
The cookie notice we can use like GitHub from https://cookie-script.com for free, we just need to be able to embed it. At the moment the embed doesn't work due to security settings.

**Comment by ngmiduc** [comment]
I am wondering what that cookie does actually. The name is _GA_

Does anyone know ?

**Comment by normanwankowski** [comment]
Google Analytics
